iosxr_iso2vbox.py

```diff
@@ -124,10 +149,10 @@ def main(argv):
     # Handle Input ISO (Local or URI)
     if re.search(':/', args.ISO_FILE):
         # URI Image
-        cmdstring = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
+        cmd_string = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
```

Comment:
'scp {0}@{1} .'.format(getpass.getuser(), args.ISO_FILE)

from what I've seen is the preferred way to do it now, not %

On 7/14/16, 3:26 PM, Richard Wellum wrote:
>> @@ -124,10 +149,10 @@ def main(argv):
>>       # Handle Input ISO (Local or URI)
>>       if re.search(':/', args.ISO_FILE):
>>           # URI Image
>> -        cmdstring = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
>> +        cmd_string = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
>
> I need a little more context or an example? What I've done is common and flake8 is happy.
>
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/ios-xr/iosxrv-x64-vbox/pull/10/files/fc29a814958e64ddff6a567916527f1c4084fe76#r70867726
>